src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -66,10 +66,20 @@ class ClaimSimulation extends Simulation {
     randomChars
   }
 
+  private def getRandomNinoDigits = {
+    // the first digit will always be in the range 1-9, the second digit always 1, the rest random
```

Comment:
I may be having a slow morning, but why is the second digit always 1 when the first can be from 1-9, so that means they have 1 child under 4 but possibly more than 1 child under 1? I'm confused....

Reply:
Maybe a longer comment is needed. I was confused to when I first saw it.

Reply:
The rules in the ChildDobGenerator are that if the first digit is greater than the second, the second digit is used for children under one instead.

Reply:
Comments added, and in the common library